pkg/models/mto_milestone.go

```diff
@@ -0,0 +1,44 @@
+package models
+
+import (
+	"time"
+
+	"github.com/google/uuid"
+)
+
+type MTOMilestoneStatus string
+
+const (
+	MTMNotStarted MTOMilestoneStatus = "NOT_STARTED"
+	MTMInProgress MTOMilestoneStatus = "IN_PROGRESS"
+	MTMCompleted  MTOMilestoneStatus = "COMPLETED"
+)
+
+type MTOMilestone struct {
+	baseStruct
+	modelPlanRelation
+
+	Name                       *string            `json:"name" db:"name"`
+	mtoCommonMilestoneRelation                    //TODO (mto) maybe we shouldn't have an mto relation here? maybe just add the field?
```

Comment:
I was going to provide similar feedback here (as is in the `TODO` comment)

I think just the ID here would be preferred since I don't think anything else in the MTO space will have a common milestone relation (unless common solutions will? but maybe just simpler to keep it just the ID here and extract it out later if we want to?)

Reply:
yeah I think that makes sense, there isn't a ton of shared logic to it. I think the only other place like you said is common solution. It probably is clearer just adding the field itself.

Reply:
removed in [36ca5a1](https://github.com/CMS-Enterprise/mint-app/pull/1454/commits/36ca5a1daa1d8d4c48cb4ea623534e2b33b1f373)